src/scripts/llm.js

```diff
@@ -24,6 +25,26 @@ async function getModelAndTokenizer(model_path) {
 }
 
 async function generateResponse(actor, input) {
+    let response;
+    let llmType = await actor.getFlag("unkenny", "llmType")
+    if ( llmType == 'api' ) {
+        response = getAPIResponse(actor, input)
+    } else if (llmType == 'local') {
+        response = getModelResponse(actor, input);
```

Comment:
Two remarks here: I think in analogy to the other case I'd prefer the function name `getResponseLocally` or something along those lines.

And this and line 31 needs to be `await getResponse(...)`. Otherwise `response` has the type `Promise`, not the type `String`.